### PR TITLE
feat: add faq components and section on signup page

### PR DESCRIPTION
**Pull request by kittybest**
**Description**
- add FAQ section on the signup page
- this is just an FAQ template, who fork this branch should modify the `FAQList` part.

### CODE REVIEW

File: src/features/signup/components/faqList.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { FAQItem } from "./faqItem";
+
+export const FAQList = (): JSX.Element => (
+  <div className="mt-28 flex flex-col items-center justify-center">
+    <h1 className="mb-8">FAQ</h1>
+
+    <FAQItem
+      description="This found is focused on supporting Mexican projects working on reducing the gender gap, collaborating with local communities, and ReFi."
```
Comment (ctrlc03) [inline src/features/signup/components/faqList.tsx:8]: typo: "round"

Comment (ctrlc03) [reply src/features/signup/components/faqList.tsx:8]: Let's remove mentions to EthMexico? "This round is focused on supporting public goods projects which made an impact on the Ethereum ecosystem" (to be more general)

File: src/features/signup/components/faqList.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    <FAQItem
+      description="To participate in this round as a project or as a voter, you must be a Mexican community member and have on-chain proofs, such as: POAPs, Unlock Protocol, NFTs or EAS."
+      title="Who are the requirements for participation?"
+    />
+
+    <FAQItem
+      description={
+        <div className="flex flex-col gap-4">
+          <p>
+            This round will be enabled by EasyRetroPGF Hypercerts EAS and MACI, Minimal Anti-Collusion Infrastructure.
```
Comment (ctrlc03) [inline src/features/signup/components/faqList.tsx:21]: there's no hypercerts

File: playwright-report/index.html
Comment (0xmad) [inline playwright-report/index.html:1]: This folder should not be commited

File: src/features/signup/components/faqList.tsx (hunk 2 above)
Comment (ctrlc03) [inline src/features/signup/components/faqList.tsx:21]: Minimal Anti-Collusion Infrastructure (MACI)

File: src/features/signup/components/faqItem.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    setIsOpen(!isOpen);
+  }, [isOpen, setIsOpen]);
+
+  return (
+    <div className="flex w-3/5 flex-col gap-4 border-b border-b-black py-6">
+      <button className="flex cursor-pointer justify-between" type="button" onClick={openDescription}>
+        <p className="font-mono text-2xl uppercase">{title}</p>
+
+        <Image
+          alt=""
```
Comment (0xmad) [inline src/features/signup/components/faqItem.tsx:20]: Please add alt

File: src/features/signup/components/faqList.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  <div className="mt-28 flex flex-col items-center justify-center">
+    <h1 className="mb-8">FAQ</h1>
+
+    <FAQItem
+      description="This found is focused on supporting Mexican projects working on reducing the gender gap, collaborating with local communities, and ReFi."
+      title="what is the focus of this round?"
+    />
+
+    <FAQItem
+      description="To participate in this round as a project or as a voter, you must be a Mexican community member and have on-chain proofs, such as: POAPs, Unlock Protocol, NFTs or EAS."
```
Comment (ctrlc03) [inline src/features/signup/components/faqList.tsx:13]: you must be whitelisted by the organizers and old either a specific NFT, POAP or an EAS attestation

File: src/layouts/BaseLayout.tsx [hunk 5]
```diff
@@ -46,6 +48,8 @@ export const BaseLayout = ({
   requireAuth = false,
   eligibilityCheck = false,
   showBallot = false,
+  marginTop = true,
+  paddingX = true,
```
Comment (0xmad) [inline src/layouts/BaseLayout.tsx:52]: Passing style props is not a good practice. Use class name instead

File: src/layouts/DefaultLayout.tsx [hunk 6]
```diff
@@ -14,7 +14,7 @@ import { EAppState } from "~/utils/types";
 
 import { BaseLayout, type LayoutProps } from "./BaseLayout";
 
-type Props = PropsWithChildren<
+type ILayoutProps = PropsWithChildren<
```
Comment (0xmad) [inline src/layouts/DefaultLayout.tsx:17]: Let's use interface instead

**Review by 0xmad — CHANGES_REQUESTED** [review]
@kittybest thanks, left some comments

File: playwright-report/index.html
Comment (kittybest) [reply playwright-report/index.html:1]: yes I just found, thanks for reminding

File: src/features/signup/components/faqItem.tsx
Comment (0xmad) [inline src/features/signup/components/faqItem.tsx:1]: File name should be FaqItem

File: src/features/signup/components/faqList.tsx
Comment (0xmad) [inline src/features/signup/components/faqList.tsx:1]: File name should be FaqList

File: src/features/signup/components/faqItem.tsx
Comment (kittybest) [reply src/features/signup/components/faqItem.tsx:1]: I renamed it, but it became to be duplicate files, let me figure it out

**Review by 0xmad — APPROVED** [review]
@kittybest thanks!

**Pull request merged**